client/src/modules/payroll/account_configuration/templates/account.tmpl.html

```diff
@@ -0,0 +1,3 @@
+<div class="ui-grid-cell-contents text-left">
```

Comment:
The ui-grid templates are usually pretty heavy to load.  Could you refactor this to simply render a value?  It would be much more performant.   Like this:
```js
.then(rows => {
  rows.forEach(row => row.hrAccountText = `[${row.account_number}] ${row.account_label}`);
  // etc...
})


// In your column configuration, you can now just use the `hrAccountText` column!
```